pcsx2/gui/AppMain.cpp

```diff
@@ -559,6 +562,9 @@ void Pcsx2App::LogicalVsync()
 				DevCon.Warning("FMV off");
 				DoFmvSwitch(false);
 				FMVstarted = false;
+				ScopedCoreThreadPause paused_core(new SysExecEvent_SaveSinglePlugin(PluginId_GS));
+				g_Conf->EmuOptions.Speedhacks.EECycleRate = eecycle_backup;
+				paused_core.AllowResume();
```

Comment:
This causes the GS plugin to be suspended and resumed twice when an FMV ends - once in `DoFmvSwitch()`, once here. It should only be suspended and resumed once.

I think restoring the original EE cyclerate value can be done in `DoFmvSwitch()`.